backend/instancer/api/account.py

```diff
@@ -336,6 +336,26 @@ def login() -> ResponseReturnValue:
     }
 
 
+@blueprint.route("/logout", methods=["POST"])
+def logout() -> ResponseReturnValue:
+    """Logout the current user
+
+    Requires body param login_token"""
+    try:
+        token = request.form["token"]
```

Comment:
Currently flask is expecting form data but the frontend is sending json as plain text. Choose one or the other - both have their merits. I'd personally make flask read the json but it's up to you

Reply:
I'll stick to JSON for simplicity.